custom/src/main/java/com/splunk/opentelemetry/SplunkConfiguration.java

```diff
@@ -34,11 +35,11 @@
     // disable otel runtime-metrics instrumentation; we use micrometer metrics instead
     config.put("otel.instrumentation.runtime-metrics.enabled", "false");
 
-    config.put("otel.traces.exporter", "jaeger-thrift-splunk");
+    config.put("otel.traces.exporter", "otlp");
     // http://localhost:9080/v1/trace is the default endpoint for SmartAgent
     // http://localhost:14268/api/traces is the default endpoint for otel-collector
     config.put(OTEL_EXPORTER_JAEGER_ENDPOINT, "http://localhost:9080/v1/trace");
-    config.put("otel.propagators", "b3multi");
+    config.put("otel.propagators", "tracecontext,baggage");
```

Comment:
this is the default, correct? So we shouldn't need to set it at all?

Reply:
Yes, it's the default value in the SDK - I thought that explicitly putting this here won't hurt, and it makes checking if we adhere to the GDI spec a little bit easier.

Reply:
That's not a bad reason; would like other thoughts on whether we should re-iterate the defaults in our configuration.

Reply:
That was the only major reason that I had.